pkg/cmd/scan.go

```diff
@@ -237,7 +239,7 @@ func scanRun(opts *pkg.ScanOptions) error {
 	globaloutput.Printf(color.WhiteString("Scan duration: %s\n", analysis.Duration.Round(time.Second)))
 
 	if !opts.DisableTelemetry {
-		telemetry.SendTelemetry(analysis)
+		telemetry.SendTelemetry(store)
```

Comment:
I think this method should not have any argument, memory store should be passed in telemetry constructor, here we should only have to do `telemetry.SendTelemetry()` imho

Reply:
Not sure to understand, how should we pass memory store to telemetry if we don't pass it to telemetry.SendTelemetry function ?

Reply:
My bad, in my mind in `telemetry.SendTelemetry`, `telemetry` was a struct initialized earlier but it is only the pkg name. forget this remark.